interface/app/$libraryId/overview/FileKindStats.tsx

```diff
@@ -1,99 +1,201 @@
-import clsx from 'clsx';
-import { motion } from 'framer-motion';
-import { useRef } from 'react';
-import { Link, useLocation } from 'react-router-dom';
-import { formatNumber, SearchFilterArgs, useLibraryQuery } from '@sd/client';
-import { Icon } from '~/components';
+import React, { useState, useEffect, useRef, useCallback, MouseEventHandler } from 'react';
+import { Card, Tooltip } from '@sd/ui';
+import { useIsDark } from '~/hooks';
+import { getIcon } from '@sd/assets/util';
+import { useLibraryQuery } from '@sd/client';
+import { useNavigate } from 'react-router';
 import { useLocale } from '~/hooks';
+import { Info } from '@phosphor-icons/react';
+import { motion } from 'framer-motion';
+
+const INFO_ICON_CLASS = "inline size-3 text-ink-faint opacity-0";
+const TOTAL_FILES_CLASS = "flex items-center justify-between whitespace-nowrap text-sm font-medium text-ink-dull mt-2 px-1";
+const UNIDENTIFIED_FILES_CLASS = "relative flex items-center text-xs text-ink-faint";
```

Comment:
We should be a little more explicit that this is a 'list' of classes

```suggestion
const INFO_ICON_CLASSLIST = "inline size-3 text-ink-faint opacity-0";
const TOTAL_FILES_CLASSLIST = "flex items-center justify-between whitespace-nowrap text-sm font-medium text-ink-dull mt-2 px-1";
const UNIDENTIFIED_FILES_CLASSLIST = "relative flex items-center text-xs text-ink-faint";
```